test/integration/controllers/admin.spec.js

```diff
@@ -0,0 +1,66 @@
+const { serverInit, serverCleanup } = require('~/test/setup')
+const {
+  roles: { ADMIN }
+} = require('~/consts/auth')
+const User = require('~/models/user')
+const Role = require('~/models/role')
+const { USER_NOT_REGISTERED } = require('~/consts/errors')
+
+describe('Admin controller', () => {
+  let app, server
+
+  beforeAll(async () => {
+    ;({ app, server } = await serverInit())
```

Comment:
do we need **';'** at the beginning of the line?

Reply:
that's prettier's auto-formatting, not sure if it's proper either